src/main/java/frc/robot/subsystems/DashboardButton.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import java.util.EnumSet;
+
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.networktables.NetworkTableEvent;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+public class DashboardButton
+{
+    private GenericEntry _entry;
+    public DashboardButton(GenericEntry entry)
```

Comment:
Let's add a line of whitespace above the constructor and below the member variable